tests/check_blas.c

```diff
@@ -0,0 +1,40 @@
+#include <stdlib.h>
+
+#include <check.h>
+
+#include "gpuarray/array.h"
+#include "gpuarray/blas.h"
+#include "gpuarray/error.h"
+#include "gpuarray/types.h"
+
+extern void *ctx;
+
+void setup(void);
+void teardown(void);
+
+#define ga_assert_ok(e) ck_assert_int_eq(e, GA_NO_ERROR)
+
+START_TEST(test_gemmBatch_3d) {
+  GpuArray A;
+  GpuArray B;
+  GpuArray C;
+
+  size_t dims[3] = {32, 32, 32};
+
+  ga_assert_ok(GpuArray_empty(&A, ctx, GA_FLOAT, 3, dims, GA_C_ORDER));
```

Comment:
I can make it that, but the problem was triggered when the kernels are compiled, not when they are run, so it doesn't matter.